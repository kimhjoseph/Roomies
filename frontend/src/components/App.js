import React from "react";
import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
import RoomiesLanding from "./RoomiesLanding";
import RoomiesHome from "./RoomiesHome";
import ShoppingList from "./ShoppingList";
import Settings from "./Settings"
import ChoreList from "./ChoreList";
import SignUp from "./SignUp";
import Login from "./Login";
import GroupCard from "./GroupCard"
import "./App.css";

function App() {
  return (
    <div>
      <header style={{ backgroundColor: "#F9F9F9" }}>
        <Router>
          <div>
            <Switch>
            <Route path="/signup">
                <SignUp />
              </Route>
              <Route path="/login">
                <Login />
              </Route>
              <Route path="/join">
                <GroupCard />
              </Route>
              <Route path="/settings">
                <Settings />
              </Route>
              <Route path="/list">
                <ShoppingList />
              </Route>
              <Route path="/home">
                <RoomiesHome />
              </Route>
              <Route path="/chores">
                <ChoreList/>
              </Route>
              <Route path="/">
                <RoomiesLanding />
              </Route>
            </Switch>
          </div>
        </Router>
      </header>
    </div>
  );
}

export default App;
