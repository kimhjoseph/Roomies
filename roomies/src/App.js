import React from 'react';
import {
  BrowserRouter as Router,
  Switch,
  Route,
  Link
} from "react-router-dom";
import RoomiesLanding from './RoomiesLanding';
import RoomiesHome from './RoomiesHome';
import './App.css';

function App() {
  return (
    <div>
      <header style={{backgroundColor: '#F9F9F9', height: '1200px'}}>
        <Router>
          <div>
            <Switch>
              <Route path="/home">
                <RoomiesHome/>
              </Route>
              <Route path="/">
                <RoomiesLanding/>
              </Route>
            </Switch>
          </div>
        </Router>
      </header>
    </div>
  );
}

export default App;
