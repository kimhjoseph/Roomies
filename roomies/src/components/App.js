import React from "react";
import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
import RoomiesLanding from "./RoomiesLanding";
import RoomiesHome from "./RoomiesHome";
import ShoppingList from "./ShoppingList";
import Settings from "./Settings"
import Chores from "./Chores";
import "./App.css";

function App() {
  return (
    <div>
      <header style={{ backgroundColor: "#F9F9F9" }}>
        <Router>
          <div>
            <Switch>
              <Route path="/settings">
                <Settings />
              </Route>
              <Route path="/list">
                <ShoppingList />
              </Route>
              <Route path="/home">
                <RoomiesHome />
              </Route>
              <Route path="/chores">
                <Chores/>
              </Route>
              <Route path="/">
                <RoomiesLanding />
              </Route>
            </Switch>
          </div>
        </Router>
      </header>
    </div>
  );
}

export default App;
