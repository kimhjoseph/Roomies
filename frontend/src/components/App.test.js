import App from './App.js';
 const renderer = require('react-test-renderer');
 const React = require('react');


test('fake test', () => {
  expect(true).toBeTruthy();
});

test('snapshot tests', () => {
	const component = renderer.create(<App />,);
	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
