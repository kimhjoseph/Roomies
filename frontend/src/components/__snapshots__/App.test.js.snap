// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot tests 1`] = `
<div>
  <header
    style={
      Object {
        "backgroundColor": "#F9F9F9",
      }
    }
  >
    <div>
      <div
        className="main"
      >
        <div
          className="inner-container"
        >
          <h1
            style={
              Object {
                "fontSize": "70px",
              }
            }
          >
            Roomies
          </h1>
          <h5
            style={
              Object {
                "margin": "0px 0px 24px 0px",
              }
            }
          >
            It's time to make your Roommates Great
          </h5>
        </div>
        <div
          className="inner-container"
        >
          <a
            href="/login"
            onClick={[Function]}
          >
            <button
              className="custom-landing-button"
            >
              Login
            </button>
          </a>
          <a
            href="/signup"
            onClick={[Function]}
          >
            <button
              className="custom-landing-button"
            >
              Signup
            </button>
          </a>
        </div>
      </div>
    </div>
  </header>
</div>
`;
